OMEdit/OMEditLIB/OMS/ElementPropertiesDialog.cpp

```diff
@@ -355,38 +351,50 @@ void ElementPropertiesDialog::updateProperties()
       if (pInterfaces[i]->causality == oms_causality_parameter) {
         QString parameterValue = mParameterLineEdits.at(parametersIndex)->text();
         parametersIndex++;
-        if (pInterfaces[i]->type == oms_signal_type_real) {
-          OMSProxy::instance()->setReal(nameStructure, parameterValue.toDouble());
-        } else if (pInterfaces[i]->type == oms_signal_type_integer) {
-          OMSProxy::instance()->setInteger(nameStructure, parameterValue.toInt());
-        } else if (pInterfaces[i]->type == oms_signal_type_boolean) {
-          OMSProxy::instance()->setBoolean(nameStructure, parameterValue.toInt());
-        } else if (pInterfaces[i]->type == oms_signal_type_string) {
-          qDebug() << "ElementPropertiesDialog::updateProperties() oms_signal_type_string not implemented yet.";
-        } else if (pInterfaces[i]->type == oms_signal_type_enum) {
-          qDebug() << "ElementPropertiesDialog::updateProperties() oms_signal_type_enum not implemented yet.";
-        } else if (pInterfaces[i]->type == oms_signal_type_bus) {
-          qDebug() << "ElementPropertiesDialog::updateProperties() oms_signal_type_bus not implemented yet.";
-        } else {
-          qDebug() << "ElementPropertiesDialog::updateProperties() unknown oms_signal_type_enu_t.";
+        if (parameterValue.isEmpty()) {
+          // delete start values only
+          OMSProxy::instance()->omsDelete(nameStructure + ":start");
+        }
+        else {
```

Comment:
Keep the `else` on the same line like `} else {` and have a space before the starting brace.

Reply:
fixed with this https://github.com/OpenModelica/OpenModelica/pull/6785/commits/5960d7d8184d97407ebd4c6b9a82866de6924120

Reply:
> Should we merge this now or do you want to add more?

We can merge this PR,